app/models/profile_field.rb

```diff
@@ -13,8 +13,7 @@ class ProfileField < ApplicationRecord
 
   enum display_area: {
     header: 0,
-    left_sidebar: 1,
-    settings_only: 2
```

Comment:
what happens to rows that have this value in the DB?

Reply:
They were already removed with a DUS here: https://github.com/forem/forem/pull/14206